test/tests/generic/test_crud_client.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 // Call within "It"
+// TestCrudClient
```

Comment:
?

Reply:
ah started to update a function comment and then got distracted.Thanks for the callout, ILl update